code/modules/mob/living/silicon/ai/say.dm

```diff
@@ -116,11 +116,11 @@ var/const/VOX_DELAY = 600
 	//writepanic("[__FILE__].[__LINE__] ([src.type])([usr ? usr.ckey : ""]) \\/mob/living/silicon/ai/verb/announcement_help()  called tick#: [world.time]")
 
 
-	var/dat = "Here is a list of words you can type into the 'Announcement' button to create sentences to vocally announce to everyone on the same level at you.<BR> \
+	var/dat = list("Here is a list of words you can type into the 'Announcement' button to create sentences to vocally announce to everyone on the same level at you.<BR> \

```

Comment:
;_;

You can encase strings in {} and include \ as line breaks and it'll work better or something, it's what N3X15's ```fixstringidiocy.py``` script changes strings to if I remember correctly.